### PR TITLE
Ignore refresh if signify is not active

### LINKED ISSUE #352: "SignifyRefresh" re-enables signify after "SignifyDisable"
`SignifyRefresh` will re-enable signify, even after `SignifyDisable`.

Repro steps:
- with signify installed and no other options specified, open a file that would have signs from signify
- `:SignifyDisable` (signs should go away)
- `:SignifyRefresh` (signs come back)

I did a git bisect, and it looks like [8de3e8](https://github.com/mhinz/vim-signify/commit/8de3e8) introduced the issue.

This bug is also a problem for the "FocusGained" autocmd event, since that (correctly) triggers a `SignifyRefresh`, re-enabling signify on buffers when changing focus between programs. https://github.com/mhinz/vim-signify/blob/c3d450eb5f5e76d99d23b10493d4e08c5bb1ae71/autoload/sy.vim#L137

Environment:
- editor:  neovim 0.4.3
- terminal: alacritty 0.4.3
- OS: NixOS 20.03

### PR DESCRIPTION
**Pull request by KapJI**
Add check if signify is enabled.
Fixes #352

### CODE REVIEW

File: autoload/sy/util.vim [hunk 1]
```diff
@@ -24,6 +24,10 @@ endfunction
 
 " #refresh_windows {{{1
 function! sy#util#refresh_windows() abort
+  " return if signify is not active
+  if empty(getbufvar(bufnr(''), 'sy'))
```
Comment (jamessan) [inline autoload/sy/util.vim:28]: `empty(get(b:, 'sy', {}))` is more inline with the code used elsewhere in Sy.

Comment (jamessan) [reply autoload/sy/util.vim:28]: Hmm, I guess not. Much of the changes in the commit that introduced this issue use `getbufvar()`.

Comment (KapJI) [reply autoload/sy/util.vim:28]: Can't agree with this. I copied this from `sy#toggle()` in `sy.vim`.
```shell
❯ rg -F 'get(b:'
❯
```
but
```shell
❯ rg -F 'getbufvar(bufnr('
autoload/sy.vim
86:  call call(empty(getbufvar(bufnr(''), 'sy')) ? 'sy#start' : 'sy#stop', [])
109:  return !empty(getbufvar(bufnr(''), 'sy'))

autoload/sy/repo.vim
260:  if empty(getbufvar(bufnr(''), 'sy'))

autoload/sy/util.vim
28:  if empty(getbufvar(bufnr(''), 'sy'))
100:  let sy = getbufvar(bufnr(''), 'sy')
```

Comment (KapJI) [reply autoload/sy/util.vim:28]: But I can change it anyway if you want to.

Comment (KapJI) [reply autoload/sy/util.vim:28]: Sorry, didn't refresh the page before sending the reply. Yeah, in most places `getbufvar` is used.

**Comment by Shane-XB-Qian** [comment]
looks that was a side-effect of 'focusgained', i noticed there was some unexpected refresh after that feature on.
// it was expected only a feature of gvim... but now tui too.
now supposed this made that through............ :-)

**Comment by Shane-XB-Qian** [comment]
um....... or perhaps no? looks this just prevent the empty sy to fresh...........

**Pull request merged**